ipalib/constants.py

```diff
@@ -215,6 +215,7 @@
     ('script', object),  # sys.argv[0]
     ('bin', object),  # The directory containing the script
     ('home', object),  # $HOME
+    ('fips_mode', object),  # FIPS-mode of the server
```

Comment:
You're right. I didn't notice these settings are global, not just test-specific. I removed this bit and fixed the test instead.